releasenotes/notes/fixed-hartree-fock-447c33a956ffea84.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Users may now bypass experiments associated with the Hartree-Fock bitstring and replace their results with a specified Hartree-Fock value using the ``hf_energy`` class field in :class:`~circuit_knitting.forging.EntanglementForgingGroundStateSolver`. Refer to the :ref:`explanatory material <Fixing the Hartree-Fock bitstring>` for more information.
```

Comment:
Will this pick up the reference from the other file? I'm reading intersphinx docs, and they say all references are global, but I'm not 100% sure. @garrison ?